service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2787,31 +2776,22 @@ components:
       properties:
         type:
           $ref: "#/components/schemas/ResourceType"
-          nullable: false
         studyId:
           $ref: "#/components/schemas/StudyId"
-          nullable: false
         studyDisplayName:
           $ref: "#/components/schemas/StudyDisplayName"
-          nullable: true
```

Comment:
What is the default for `nullable` -- is it different for required/not properties?

Reply:
Default value of nullable = true. Same for required / not required. 
Reason for removing here -- nullable applies to primary types and direct object definitions. 
Here again, studyDisplayName refers to a definition  '$ref: "#/components/schemas/StudyDisplayName"'

Reply:
Understood, okay thanks for explaining. I clearly did not grok all the details of nullable/required/description as I was writing this file, so removing these is fine for this PR and I'll add a new ticket to review all these properties for the whole file.

Reply:
Thanks, Lmk if you want me to create a ticket for properties review. Constraints like min/max, string length etc may need to be reviewed as well

Reply:
Created https://verily.atlassian.net/browse/BENCH-2872.
Good idea about constraints, will note that also.